src/coffee/run.coffee

```diff
@@ -125,6 +125,7 @@ module.exports = class
       .option '--publish', 'When given, all changes will be published immediately'
       .option '--updatesOnly', "Won't create any new products, only updates existing"
       .option '--dryRun', 'Will list all action that would be triggered, but will not POST them to SPHERE.IO'
+      .option '--defaultState', "When given, specifies which state to assign imported products to, if they don't have one"
```

Comment:
Maybe add also info that the user should provide a state **key**?

Reply:
Also, add this new variable also to readme - https://github.com/sphereio/sphere-node-product-csv-sync/#usage

Reply:
Done.